src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -94,6 +100,39 @@ private static IEnumerable<CredentialCacheCount> GetCredentialCacheCounts()
             yield return CreateHostPortCredentialCacheCount();
             yield return CreateUriAndHostPortCredentialCacheCount();
         }
+        
+        public static IEnumerable<object[]> StandardAuthTypeWithNetworkCredential =>
+            new[]
+            {
+                new object[] {authenticationTypeNTLM, credential1},
+                new object[] {authenticationTypeKerberos, credential2},
+                new object[] {authenticationTypeNegotiate, credential3},
+                new object[] {authenticationTypeBasic, credential4},
+                new object[] {authenticationTypeDigest, credential5},
```

Comment:
You don't really need to have credential1 thru credential5 separate objects.  All of them are using the same username/password.  So, you can simply have a single NetworkCredential object (i.e. customCredential) and use it in all of the new object[] places you have.  It is ok to share the same network credential object like that.